beta-src/src/components/map/components/WDTerritory.tsx

```diff
@@ -1,6 +1,13 @@
 import { useTheme } from "@mui/material";
 import * as React from "react";
+import countryMap from "../../../data/map/variants/classic/CountryMap";
```

Comment:
Is this looking for the root path? can we have an env variable that can help us with this? so that we don't have to repeat `../../../`

Reply:
VSCode automatically imports files like that, in this case using an .env might be more work as the paths would have to be written out manually.